server/routes/testutils/testData.ts

```diff
@@ -156,4 +157,10 @@ export default class TestData {
     reference = '-afe~dcb~fc',
     incentiveLevels = ['ENHANCED'],
   }: Partial<IncentiveLevelGroup> = {}): IncentiveLevelGroup => ({ name, reference, incentiveLevels })
+
+  static createIncentiveGroupDto = ({
+    name = 'Wing A',
```

Comment:
Might want to put a more realistic name - only test data but could end up confusing us 😅!

Reply:
what would you suggest? same naming pattern as location group

Reply:
Something like:
```suggestion
    name = 'Enhanced prisoners',
```

Reply:
oh.... I thought you mean the variable name, not the value of name 😂